src/commands/role.ts

```diff
@@ -69,6 +73,58 @@ class RoleCMD extends Command {
         message.channel.send(msg);
     }
 
+    // Remove role flag
+    private async removeRoles(
+        guild: Guild,
+        message: Message,
+        bot: ClientUser,
+        arg: string
+    ): Promise<void> {
```

Comment:
I can see the similar props being used on two methods `listRoles` & `removeRoles`, we can use interface here and extend the `arg` for `removeRoles` to keep it clean.

Reply:
sure